index.html

```diff
@@ -24,7 +24,7 @@
         <script src="./node_modules/react-dom/dist/react-dom.js"></script>
 
         <!-- Main -->
-        <script src="./dist/bundle.js"></script>
+        <script src="./dist/librarie.min.js"></script>
```

Comment:
The `min` will only be created if we run `npm run build`, if we just run `webpack` or `npm run dev`, only librarie.js will be created, but I am not sure which one we should put here.